### PR TITLE
[apptools-windows] Add tests to test 'check' command for apptools windows

### PR DESCRIPTION
**Pull request by yunxliu**
Impacted tests(approved): new 2, update 1, delete 0
Unit test platform: [Android]
Unit test result summary: pass 3, fail 0, block 0

https://crosswalk-project.org/jira/browse/XWALK-4949

### CODE REVIEW

File: apptools/apptools-windows-tests/apptools/check_host.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        os.chdir(comm.XwalkPath)
+        cmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app check"
+        (return_code, output) = comm.getstatusoutput(cmd)
+        self.assertEquals(return_code, 0)
+        self.assertNotIn("ERROR:", output[0])
+
+    def test_check_host_android(self):
+        comm.setUp()
+        os.chdir(comm.XwalkPath)
+        cmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app check android"
```
Comment (Honry) [inline apptools/apptools-windows-tests/apptools/check_host.py]: Why we have to check android platform prerequisite in windows backend testing?

File: apptools/apptools-android-tests/apptools/check_host.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def test_check_host_without_platforms(self):
         cmd = comm.HOST_PREFIX + comm.PackTools + "crosswalk-app check"
         (return_code, output) = comm.getstatusoutput(cmd)
         self.assertEquals(return_code, 0)
-        self.assertNotIn("ERROR:", output[0])
+        self.assertNotIn("ERROR:", output[0].split('target windows')[0])
```
Comment (Honry) [inline apptools/apptools-android-tests/apptools/check_host.py]: I suppose target platform should be android.

Comment (yunxliu) [inline apptools/apptools-android-tests/apptools/check_host.py]: output[0].split('target windows')[0] is the output of testing android host setup. It is used to check  android host setup

Comment (Honry) [inline apptools/apptools-android-tests/apptools/check_host.py]: My mistake, LGTM

**Pull request merged**